src/surge-xt/osc/OpenSoundControl.cpp

```diff
@@ -556,16 +558,18 @@ void OpenSoundControl::oscMessageReceived(const juce::OSCMessage &message)
                 sspPtr->oscRingBuf.push(SurgeSynthProcessor::oscToAudio(p, val));
             }
         }
-
-#ifdef DEBUG_VERBOSE
-        std::cout << "Parameter OSC name:" << p->get_osc_name() << "  ";
-        std::cout << "Parameter full name:" << p->get_full_name() << std::endl;
-#endif
     }
 
     // Patch changing
     else if (address1 == "patch")
     {
+        if (querying)
+        {
+            std::string patchpath = synth->storage.patch_list[synth->patchid].path.u8string();
```

Comment:
Ahh unfortunately there's some cases like in drag and drop where patched can be -1 so you need to check this here I think and return an unknown

so if synth->patched >= 0 && s->pid < s->storage.patch_list.size() then....

Reply:
OK, not sure how to trigger this, but it's handled.

Reply:
You would find a random fxp and drag and drop it onto your surge

then it would be loaded but not in the patch list